sklearn/linear_model/ridge.py

```diff
@@ -526,6 +526,8 @@ class RidgeClassifier(LinearClassifierMixin, _BaseRidge):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        Note that regressors are always centered no matter what and no
```

Comment:
Thank you guys.

"If True, the regressors X will be normalized before regression by scaling them
(dividing them by their standard deviation). This process is incorporated in the model and no
manual normalization of X is required before using the model at the time of prediction.
Note that centering (subtracting the mean of the regressors) is always performed
even with normalize=False, and also incorporated in the model."

might be, I hope, clear enough, albeit a bit longer.